apps/studio/src/lib/data/tools.ts

```diff
@@ -42,7 +42,7 @@ export const Mutators = {
    */
   genericMutator(value: any, preserveComplex: boolean = false): JsonFriendly {
     const mutate = Mutators.genericMutator
-    if (_.isBuffer(value)) return value.toString()
```

Comment:
The return value showed incorrect representation of the real buffer. We need to use a specific encoding like hexadecimals for example.

Reply:
Probably toString(16)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString

Reply:
Yeah hex is the right way to do it. Although probably we need to prefix hex strings with `s/` when displayed (although not edited), this is a convention used in many UIs for hex'd binary values